docker-build.sh

```diff
@@ -5,10 +5,10 @@ set -euo pipefail
 cd "$(dirname "${BASH_SOURCE[0]}")"
 . stack-helpers.sh
 
-[ $# -eq 1 ] || abort usage: $(basename "${BASH_SOURCE[0]}") STACK
+[ $# -eq 2 ] || abort usage: $(basename "${BASH_SOURCE[0]}") STACK
```

Comment:
`BUILD.md` needs updating to reflect two args must now always be passed. Or alternatively, `IMAGE_TAG` below could have a fallback default.

Reply:
Thanks. I'll update that doc.